### PR TITLE
Week 1 Paragraphs about Me

### PR DESCRIPTION
**Pull request by rqtrinh**
Add paragraphs of biography, experience, and me! Add headers for organization.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,8 +8,34 @@
...
+      <p>My name is Ryan Trinh. I am a third year Computer Science major
+          at Cal Poly Pomona. My goal is to become a full stack developer 
+          after graduating from University. I am looking for opportunties
+          to expand and improve my current skill set. As of right now, 
+          I am well versed with Java, Python, Data Structures, and OOP.
+          I am also trying to learn HTML, CSS, and Javascript which will 
+          be used for creating this portfolio.
+      </p>
+      <h2>Experiences</h2>
+      <p>I have had many experiences in my career. My first experience is
```
Comment (RyanSCollins) [inline portfolio/src/main/webapp/index.html]: Nitpick: tense, "is"->"was"

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,8 +8,34 @@
...
+      </p>
+      <h2>Experiences</h2>
+      <p>I have had many experiences in my career. My first experience is 
+          working as a Back End Developer at the Northrop Grumman Collaboration
+          Project. In this project, I created a program that read input controls 
+          from either a keyboard or controller. I then formatted the data into a 
+          JSON file with floating point values for directional attributes and 
+          buttons. The file was then sent to a UGV for use. During this summer, 
+          I am currently participating in the Google SPS program. I am also
+          intering at Avanade as a Full Stack Developer.
```
Comment (RyanSCollins) [inline portfolio/src/main/webapp/index.html]: Nit: spelling/typo, "interning"

**Review by RyanSCollins — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (rqtrinh) [reply portfolio/src/main/webapp/index.html]: Fixed grammar mistake in new commit

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (rqtrinh) [reply portfolio/src/main/webapp/index.html]: Fixed spelling mistake in new commit

**Review by RyanSCollins — APPROVED** [review]

**Pull request merged**